scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -361,7 +387,9 @@ function Get-SecurityDaemon {
                     -Delete ([ref] $deleteMobyEngineArchive)
             Expand-Archive $mobyEngineArchivePath $MobyInstallDirectory -Force
 
-            New-Item -Type Directory $MobyDataRootDirectory | Out-Null
+            if (-not (Test-Path $MobyDataRootDirectory)) {
```

Comment:
A simpler option is to use the `-Force` flag on the `New-Item` command. We're already using that approach when we create the domain socket parent folders.

Reply:
Whoops. Forgot about that.